docs/developer/README.md

```diff
@@ -81,6 +81,19 @@ The endpoint /admin can be used to view and manage site content, including but n
 
 5. In the browser, navigate to /admins. To verify that all is working correctly, under "domain applications" you should see fake domains with various fake statuses.
 
+## Adding to CODEOWNERS (optional)
+
+The CODEOWNERS file sets the tagged individuals as default PR reviewers on any commit regarding the content specified. An asterisk indicates a user will be added as PR reviewer for all files in the repo, while "subfolder/*" indicates a particular folder for which a user(s) will be added as default PR reviewer. To add yourself as code owner for all files:
```

Comment:
I think a link to the [CODEOWNERS docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) is a great idea. Its something we often have to look up/reference anyways, so might as well make it easy. 

I'll make the update(s) since Alysia will be out the rest of this week

Reply:
The other reason I think the docs are important to link to is that there are all sorts of nuances with order (order is really important in a codeowners file), and rather than repeating what's in the official docs I prefer linking to them